lib/customer.rb

```diff
@@ -0,0 +1,16 @@
+class Customer
+
+  attr_reader :name, :tel_no
+
+  def initialize name, tel_no
+    @name = name
+    @tel_no = tel_no
+  end
+
+  def check_bill restaurant
+    # should check restaurant orders for order from this customer and return bill for further processing inside this instance
```

Comment:
Metrics/LineLength: Line is too long. [125/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)